js-plugins/mcp/src/index.ts

```
@@ -0,0 +1,79 @@
+  const cursorMcpOptions = typeof updateCursorMcpJson == 'boolean'
+    ? { enabled: updateCursorMcpJson }
+    : updateCursorMcpJson
+
+  const vscodeMcpOptions = typeof updateVSCodeMcpJson == 'boolean'
+    ? { enabled: updateVSCodeMcpJson }
+    : updateVSCodeMcpJson
+  let farmConfig: ResolvedUserConfig = {}
+  return {
+    name: 'mcp',
+    configResolved(config) {
+      config = config
```

Biome (1.9.4)

[error] config is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)